vignettes/estimate_static_severity.Rmd

```diff
@@ -136,24 +136,24 @@ As such, we retrieve such a distribution from the literature [@barry2018]; this
 
 ### Estimating incidence of cases (or similar time-series) with a known death outcome
 
-The function `known_outcomes()` from the _cfr_ package can estimate the number of cases which have a known death outcome over time.
+The function `estimate_outcomes()` from the _cfr_ package can estimate the number of cases which have a known death outcome over time.
 
-The resulting data frame contains two new columns, "known_outcomes", for the number of known death outcomes expected for each day, and "u_t", the under-reporting factor that estimates what proportion of cases have not been reported, given the estimated number of known death outcomes.
+The resulting data frame contains two new columns, "estimated_outcomes", for the number of known death outcomes expected for each day, and "u_t", the under-reporting factor that estimates what proportion of cases have not been reported, given the estimated number of known death outcomes.
```

Comment:
is it the proportion that has been or that hasn't been ascertained?

Reply:
Updated to be the same as the documentation.